static/scss/homepage.scss

```diff
@@ -208,13 +212,32 @@ body.app-media {
 }
 .app-media .media-list.is-grid .media-item {
   position: relative;
-  width: 95%;
   padding: 0;
   margin-bottom: 18px;
   cursor: pointer;
   border: 1px solid #DEDEDE;
   border-radius: 3px;
   box-shadow:0 1px 2px rgba(43,59,93,.12),0 0 13px rgba(43,59,93,.12);
+  min-height: 410px;
+
+  &.placeholder-card {
+    background: #393939;
+    background-image: url(/static/images/part-globe.png);
+    background-repeat: no-repeat;
+    background-position: center bottom;
+    &:hover {
+      background: #393939;
+      background-image: url(/static/images/part-globe.png);
+      background-repeat: no-repeat;
+      background-position: center bottom;
+    }
+    &:active {
+      background: #393939;
+      background-image: url(/static/images/part-globe.png);
+      background-repeat: no-repeat;
+      background-position: center bottom;
+    }
+  }
```

Comment:
yes, as per design i can see there is not hover or selected mode. This is to override card behavior because when we it become when splash.